modular_skyrat/modules/encounters/code/nri_raiders.dm

```diff
@@ -195,14 +228,31 @@ GLOBAL_VAR(first_officer)
 	// if this is the first officer, keep a reference to them
 	if(!GLOB.first_officer)
 		GLOB.first_officer = spawned_human
-
+		to_chat(spawned_human, "<B>With you being the leader of the group and having a special designation, 'Actual', it's your duty to make sure this entire operation \
+		goes smoothly. As in, doesn't result in an intergalactic political scandal, or an unneecessary shooting. It's also very likely expected for you to be performing \
+		all the necessary negotiations, so do prepare yourself for that.")
+	to_chat(spawned_human, "<B>The station has overriden the response system for the reasons unknown, keep the ship intact, communicate with the station, \
+	perform an inspection to determine the legitimacy of the fine, and try to get the funds yourself, if it's legitimate. \
+	In any case, perform your predefined duties and uphold some semblance of intergalactic law and professionalism, even if just for show.</B> <BR><BR>\
+	Also, a small OOC clarification: none of your objectives are meant to be completable mechanically, so don't stress yourself over not greentexting or anything; \
+	If you have a better plan than 'completing' them, like an idea for a gimmick, it's better to communicate with the admins and your colleagues to possibly allow you to \
+	do something custom.")
 	apply_codename(spawned_human)
 
 
 /obj/effect/mob_spawn/ghost_role/human/nri_raider/officer/post_transfer_prefs(mob/living/carbon/human/spawned_human)
 	. = ..()
 	apply_codename(spawned_human)
 
+/obj/effect/mob_spawn/ghost_role/human/nri_raider/officer/equip(mob/living/carbon/human/spawned_human)
+	. = ..()
+	var/obj/item/card/id/advanced/card = spawned_human.get_idcard()
+	if(GLOB.first_officer == spawned_human)
+		card.assignment = NRI_LEADER_JOB_LIST
+	else
+		card.assignment = NRI_JOB_LIST
```

Comment:
```suggestion
		card.assignment = pick(NRI_LEADER_JOB_LIST)
	else
		card.assignment = pick(NRI_JOB_LIST)

```

Reply:
No reason to as there's already a pick proc in the define itself.